setup.py

```diff
@@ -31,7 +31,7 @@
         'wrapt',
         'tornado>=4.1,<5',
         'contextlib2',
-        'opentracing>=1.1,<2',
+	'opentracing==2.0.0',
```

Comment:
Hi @carlosalberto,
Please use spaces instead of `tab`.

Hello @yurishkuro,
I suppose that this `tab` came from [here](https://github.com/yurishkuro/opentracing-python-instrumentation/commit/8e22b09ff802860fa9d8e83f3df1409cc6b10b96#r31397576).